gym/envs/nysaq/tests/assets/rbmConfig1bt

```diff
@@ -24,7 +24,7 @@
     "name": "bt",
     "num_blockers": 1,
     "grid_y": 5,
-    "positive_reward": 1,
-    "negative_reward": 0
+    "positive_reward APPARENTLY NOT USED": 1,
```

Comment:
Can we actually use these instead? we can rename them:
win_reward: For winning. currently 5.
play_reward: For taking each action. currently 0.

Reply:
Implemented